### ISSUE TITLE
Mapping the lingering memories of the Brazilian dictatorship [Project]

### ISSUE THREAD

**Issue by RodrigoMenegat** [jsoma/data-studio-projects#11]
# Pitch
Mapping the lingering memories of the Brazilian dictatorship

## Summary
I want to map all the streets in Brazil that were named after criminals of the military dictatorship, as well as their victims.

## Details
Brazil lived under a bloody dictatorship for more than 20 years, between 1964 and 1985. A military junta governed the country, persecuting opponents and committing serious crimes against human rights. Unlike their Latin neighbors, the Brazilian government hasn’t investigated these crimes in depth until 2011, when a truth commission was created. Even though 377 people were accused of those crimes in 2014, many street names and landmarks still pay homage to them. I want to map all those places, as well as the significantly fewer locations that remember those who were killed or tortured during these “lead years”. 

This discussion is pertinent since the nations hasn’t come to terms with its troubled past yet. In the aftermath of the 2015 elections and the following economic/political crisis, many groups took to the streets demanding another military intervention. One of the leading candidates for the upcoming 2018 election is a captain of the military and frequently defends the authoritarian regime in his speeches. 

Is there any relation between this phenomenon and the silent homages that persist in the streets of the country?

**Possible headline(s):** 
- In Brazilian streets, the dictatorship still lives
- Homages to dictatorship criminals still linger in the streets of Brazil

**Data set(s):** 
- OPEN STREET MAPS SHAPEFILES - Probably mine is an older version
http://download.geofabrik.de/south-america/brazil-latest-free.shp.zip
- List of criminals made by the Brazilian Truth Comission
- List of victims made by the Brazilian Truth Comission

**Code repository:** 
https://github.com/RodrigoMenegat/studio-projects/tree/master/code/brazil-dictatorship-streets
	
**Possible problems/fears/questions:** 
Data is not standardized. There are a lot of typos, homonims and spelling variations.

## Work so far
- Filtered the Open Street Map data,
- Manually checked the streets with the name of criminals/victims to avoid mistakes and only kept the street names that were more likely than not to refer to the right person
- Created two preliminary maps
![missing_killed](https://user-images.githubusercontent.com/28814352/28273281-ecd5903c-6adb-11e7-88ec-1292993e5c43.png)

![criminals](https://user-images.githubusercontent.com/28814352/28273280-eccfaa50-6adb-11e7-84ef-9981e16b4d91.png)

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [x] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue issue has `[Project]` in the title
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far
- [x] I have received two comments of peer feedback on my pitch issue

**Comment by amandolinjames** [comment]
I like this idea and think it is interesting! I think you could simplify it to look at the most recent dictators, so the map is not so dense, but is very user-friendly.

**Comment by cathrinesot** [comment]
Very interesting! I just want to see somehow one image of a street and a summary of the dictator's story.

**Comment by RodrigoMenegat** [comment]
# Update

## Images
![mapprint](https://user-images.githubusercontent.com/28814352/28450246-0fd78e2a-6db4-11e7-82f4-a2d86a209ad8.png)
![brazil-maps](https://user-images.githubusercontent.com/28814352/28450053-c271702a-6db2-11e7-86f0-8758186701ff.png)
![major-cities](https://user-images.githubusercontent.com/28814352/28450052-c271322c-6db2-11e7-89ae-be77b3f582e9.jpg)
	
## Content
- I have created an interactive Leaflet map that allows the re
ader to navigate through the country, click on a street and read a short bio of the person it's named after. Not all bios are ready, though, and the looks still need some tweaking. Ideally, I would like to show pictures of the people as well, and include a searchbox. (In Portuguese)
> http://bit.ly/2ugDARy

- I started exploring the data in order to build a more guided narrative and drew some sketches for each major city.

## Any changes in direction or topic?
- Not so far.	
## Problems/Questions
- I am having problems with the quality of my data. Many names have different spellings and other standardization inconsistencies. I need to find a way to get around that.
- It's hard the count the total number of streets since many of them are represent by more than one geometry in the dataframe. I could group most of them using geopandas, buffers and intersects, but some really big roads are still counted more than once. They are few, though, and maybe I can manually change them.
- I have a lot of common names and homonims. I can't be sure if those streets refer really to the people who were involved in the dictatorship events. Should I keep them? Should I ignore them?

## Checklist
	
- [x] I have included my visuals
- [x] I have added writing about my topic
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository
- [x] I have received two comments of peer feedback on my project update

**Comment by cathrinesot** [comment]
Very nice! Can we se a photo of something? A dictator, a victim or something like that?

**Comment by sarahslo** [comment]
This is a really unique idea, very compelling. I think however what we are seeing here is that the data themselves are not very visual when you map them. Your instincts are completely right to zoom in to smaller areas -- I would suggest you take this a step further and look for the most complete dataset in all of this and zoom way in. Zoom in to Jova Rural at the street level! That is what this whole things is about, so take us all the way down to the streets.

You are making an excellent attempt at small multiples, but since there is not much data the overall impression we are left with is empty maps. If you just looked at the few clusters where you have good data and laid out all the names and the history of the people, with photos, or photos of the streets themselves that would be very powerful.

I like the country maps at the top. Perhaps that is one map with two levels of data on it so we have a little more density. 

This is very interesting. I'd love to simply see lists or tables of the names themselves and see if there is a frequency by person. Have you done this analysis?

Nice work.

**Comment by RodrigoMenegat** [comment]
# Final

## Project visuals/text
Here follows an HTML with the final story + graphics:
https://rodrigomenegat.github.io/studio-projects/streets-dictatorship/

## Details

**Headline:** In Brazilian streets, memories of the dictatorship still linger

**Published website version:** https://rodrigomenegat.github.io/studio-projects/streets-dictatorship/

**Code repository:** https://github.com/RodrigoMenegat/studio-projects/tree/master/code/brazil-dictatorship-streets

**Final data set(s):**  https://github.com/RodrigoMenegat/studio-projects/blob/master/code/brazil-dictatorship-streets/final-version/streets.geojson

## What did you find to be the most difficult part of this project?
After all, the OpenStreetMap data wasn't that good. There were a lot of incosistencies in spelling and duplicate entries that made the cleaning proccess a major pain, with a lot of manual work.

Also, my database had a lot of common names and I can't be sure if the streets refer specifically to the people involved in the events of the Dictatorship. Nevertheless, this issue does not affect the overall picture, and I decided to keep those in with a disclaimer in the text.

The biggest problem, though, was that there were many linestring geometries for each street in the shapefiles. This made it very hard to count the total number of streets named after each historical figure. 

In the end, I wasn't feeling confortable with the analysis I made in relation to this, even after spending a considerable ammount of time trying to treat all the lines for each street as a single polygon. Because of that, this results are not in my final story. 

This killed a lot of important angles for the material: who are the most popular criminals, dictators and victims? Which city has the most streets named after dictators or resistance fighters? I wasn't able to answer those questions.

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
I really like the interactive map, since I think there is plenty of things for the reader to explore by himself - something I like to do in my projects. Nevertheless, it's only avaliable in portuguese right now! I thought I would have time to make an English version, as well... Shame on me.

The narrative part is terribly lacking, sadly. I could not do any actual quantitative analysis. I wish I could have, at least, counted the number of streets and discovered the most 'popular' people of that historical period. 

Finally, I belive there is room for a nice qualitative approach at the topic, and I laid the foundations for that... But with no interviews or actual on the ground reporting, it's hard to call this a finished story.

## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by RodrigoMenegat** [comment]
@cathrinesot and @sarahslo: I agree with both of you and I wanna do all those things that you suggested! 

About the pictures, I'd love to add them for this 'final' version, but there are more than 700 hundred people officialy listed as victims and criminals. Too many images, too little time! :(

Sarah: sadly I couldn't come up with a list of the most common names and places, and I explain why in this last update: OpenStreetMap data is really, really, reeeeally messy. I actually did some work on that, but wasn't feeling secure at all about the results, so I decided to drop it for now. But that would be the most interesting part of the analysis, I believe!

About the empty city maps, I think that maybe I could plot all the other streets, those that aren't named after criminals or victims, in a very light and discrete color. Perhaps this would make the maps too dense with unimportant information, but they would surely be more visually pleasing. Maybe I'll do that soon and see how it works!
